simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.simulator;
+
+import com.hedera.block.simulator.generator.BlockStreamManager;
+import com.swirlds.config.api.Configuration;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.inject.Inject;
+
+/** BlockStream Simulator App */
+public class BlockStreamSimulatorApp {
+
+    private static final System.Logger LOGGER =
+            System.getLogger(BlockStreamSimulatorApp.class.getName());
```

Comment:
I think you can change this LOGGER declaration to: `private final Logger LOGGER = System.getLogger(getClass().getName());`

Reply:
will do, thanks for the suggestion.

Reply:
It does not work for static context or objects. 😅

Reply:
Ah, ok...thanks for taking a look